cypress/e2e/new-suggestion/new-suggestion.cy.ts

```diff
@@ -27,66 +27,87 @@ const instrumentsAndReviewArea = [
 ]
 
 describe("when creating a new suggestion", () => {
-  beforeEach(() => {
-    cy.login()
-    cy.visit(path)
-    // Wait so content can render properly and set up submit events
-    cy.wait(500)
+  describe("when loading instruments fails", () => {
+    beforeEach(() => {
+      cy.login()
+      cy.intercept("GET", "/rest/v1/instrument?select=*&order=instrument_name.asc", {
+        forceNetworkError: true,
+      })
+      cy.visit(path)
+      // Wait so content can render properly and set up submit events
+      cy.wait(500)
+    })
+
+    it("should error if it can't retrieve instruments", () => {
+      cy.data("failed-fetching-instruments").should("be.visible")
+    })
   })
```

Comment:
I don't think beforeEach is necessary since there is only one test in here. You could perhaps just move the code to the it block

Reply:
beforeEach() describes the steps to mock "when loading instruments fails", even if the describe() has one test, putting it all into one test makes the it() clause less isolated. If somebody forgets to update the intercept step to a new route when needed. "should error if it can't retrieve instruments" shouldn't fail because the step to get to that clause fails.